nvim/lua/configs/nvim-lsp.lua

```diff
@@ -115,6 +115,7 @@ nvim_lsp["gopls"].setup {
        symbolMatcher = "fuzzy"
      },
    },
+}
```

Comment:
I don't think this is correct. If I put that bracket there we close the setup function too soon

Reply:
No. Your lua server is now getting into gopls. Check if "{" "}" is paired!